contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
 
 Implements the IBM Security Advisor pipeline
 
+The security advisor filters the flow data obtained from skydive, performs a data transformation, and saves the information to an object store in JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+We use the term "pipeline" to describe the sequence of operations performed on the data: classify, filter, encode, store.
+In the future, we plan to have numerous pipelines performing different kinds of operations.
 
-## Setup Skydive
+## Deploy Skydive
 
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
+
+### All-in-one version
 
 ```
 make
 cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
```

Comment:
remove this line - we can simply provide an end-to-end sequence that works (without discussing deviations from default too much)